spec/requests/api/automation_requests_spec.rb

```diff
@@ -192,4 +192,40 @@
       expect(response).to have_http_status(:ok)
     end
   end
+
+  context 'Tasks subcollection' do
+    let(:automation_request) { FactoryGirl.create(:automation_request) }
+
+    it 'redirects to request_tasks subcollection' do
+      task = FactoryGirl.create(:miq_request_task, :miq_request_id => automation_request.id)
+      api_basic_authorize
+
+      run_get("#{automation_requests_url(automation_request.id)}/tasks")
+
+      expect(response).to have_http_status(:moved_permanently)
+      expect(response.redirect?).to be_truthy
```

Comment:
I think this expectation is redundant - the one above checks that it is a 301, this one checks that it is [in the set of many redirect codes](https://github.com/rack/rack/blob/master/lib/rack/response.rb#L132)